base/memory/memory_pressure_monitor_endless.cc

```diff
@@ -52,6 +61,21 @@ MemoryPressureMonitor* MemoryPressureMonitor::Get() {
       base::MemoryPressureMonitor::Get());
 }
 
+void MemoryPressureMonitor::ParseCommandLineParameters() {
+  base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
+  std::string switch_str;
+  double value;
+
+  // Parse the switch for the MODERATE threshold (should be a positive integer between 0 - 100).
```

Comment:
Nitpick: Is this comment correct, or is it a copypasta?

Reply:
OMG! Copy&pasta completely, sorry. To be tackled along with the next rebase to chromium 67